liblll/CompilerState.cpp

```diff
@@ -65,7 +65,7 @@ void CompilerState::populateStandard()
 	"(def 'makeperm (name pos) { (def name (sload pos)) (def name (v) (sstore pos v)) } )"
 	"(def 'permcount 0)"
 	"(def 'perm (name) { (makeperm name permcount) (def 'permcount (+ permcount 1)) } )"
-	"(def 'ecrecover (r s v hash) { [0] r [32] s [64] v [96] hash (msg allgas 1 0 0 128) })"
+	"(def 'ecrecover (hash v r s) { [0] hash [32] v [64] r [96] s (msg allgas 1 0 0 128) })"
```

Comment:
No, it's not a re-ordering, it's just a relabelling and is not breaking at all since the order is changed within the macro body as well. The argument labels are just documentation, and they are currently quite wrong and misleading.

Any existing code that does `(ecrecover a b c d)` will continue to translate to `{ [0] a [32] b [64] c [96] d (msg allgas 1 0 0 128) }` just as before.